configs/nosite-x86_64/variables.yaml

```diff
@@ -4,9 +4,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 variables:
-  batch_time: ''
-  mpi_command: 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe'
-  batch_submit: '{execute_experiment}'
-  batch_nodes: ''
-  batch_ranks: ''
-  batch_timeout: ''
+  mpi_command: 'mpirun -n {n_ranks} --oversubscribe'
+  n_ranks: '7'
+  n_nodes: '7'
```

Comment:
lets leave default n_nodes as 1 (single node).

Reply:
Wherever I put "7" in this file, it means "ignore this value", any other value is taken to mean "this value is set, do not change it". I assume "1" is a value we would want.

Reply:
Would it be easier to find the values that weren't set if we set them to -1 initially (an invalid value by our definition of "positive integers")?

Reply:
I was likewise hoping I could set them to something that was more obviously wrong, but Ramble expects them to be positive integers. I'll double check though to make sure.

Reply:
I have updated with new logic so we don't have to use some random number as a placeholder:

* There is now a `max_request` variable (which defaults to 1000 if unset)
* Requests above this are considered placeholders

Later on, the logic which fills in all unset variables also makes use of `max_request` to make sure it didn't accidentally request too much.

This particular thread I'll mark as resolved, because I think it's less confusing now.